tf2onnx/onnx_opset/tensor.py

```diff
@@ -1222,36 +1265,86 @@ def version_1(cls, ctx, node, **kwargs):
+            const_one_one = ctx.make_const(utils.make_name(node.name + "_const_one_one"),
+                                           np.array([1, 1], dtype=np.int64))
+            padded_block_shape = ctx.make_node('Pad',
+                                               [block_shape.output[0], const_one_one.output[0], const_one.output[0]])
+            shape_x = ctx.make_node('Shape', [padded_input_x.output[0]])
+            new_shape_x = ctx.make_node('Div', [shape_x.output[0], padded_block_shape.output[0]])
+            first_row_new_shape_x = GraphBuilder(ctx).make_slice(
+                {"data": new_shape_x.output[0], "ends": [2], "starts": [0]})
+            block_size = GraphBuilder(ctx).make_slice({"data": block_shape.output[0], "ends": [1], "starts": [0]})
+            new_fisrt_row_new_shape_x = ctx.make_node('Concat', [first_row_new_shape_x, block_size], {'axis': 0})
+            second_row_new_shape_x_first_half = GraphBuilder(ctx).make_slice(
+                {"data": new_shape_x.output[0], "ends": [3], "starts": [2]})
+            second_row_new_shape_x_second_half = GraphBuilder(ctx).make_slice(
+                {"data": new_shape_x.output[0], "ends": [4], "starts": [3]})
+            new_second_row_new_shape_x_first_half = ctx.make_node('Concat',
+                                                                  [second_row_new_shape_x_first_half, block_size],
+                                                                  {'axis': 0})
+            new_second_row_new_shape_x = ctx.make_node('Concat', [new_second_row_new_shape_x_first_half.output[0],
+                                                                  second_row_new_shape_x_second_half], {'axis': 0})
+            new_shape_x_v2 = ctx.make_node('Concat',
+                                           [new_fisrt_row_new_shape_x.output[0], new_second_row_new_shape_x.output[0]],
+                                           {'axis': 0})
+            new_x = ctx.make_node('Reshape', [padded_input_x.output[0], new_shape_x_v2.output[0]])
+            transposed_new_x = ctx.make_node('Transpose', [new_x.output[0]], {'perm': [2, 4, 0, 1, 3, 5]})
+            block_size_prod = ctx.make_node('Mul', [block_size, block_size])
+            const_zero_three = ctx.make_const(utils.make_name(node.name + "_const_zero_three"),
+                                              np.array([0, 3], dtype=np.int64))
+            padded_block_size_prod = ctx.make_node('Pad', [block_size_prod.output[0], const_zero_three.output[0],
+                                                           const_one.output[0]])
+            new_shape_x_v3 = ctx.make_node('Mul', [new_shape_x.output[0], padded_block_size_prod.output[0]])
+            ctx.remove_node(node.name)
+            new_x_v2 = ctx.make_node('Reshape', [transposed_new_x.output[0], new_shape_x_v3.output[0]], name=node.name,
+                                     outputs=node.output)
```

Comment:
What's the summarized logic here (comments would be helpful)? There are ~20 new nodes -- can they be condensed?

Reply:
Commented.